### PR TITLE
feat: package updates served with brute force

### PR DESCRIPTION
**Pull request by krysopath**
be sure to run tests xD

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,3 +9,9 @@ test:
 
 run:
 	go run ./cmd/...
+
+update:
+	go list -m -u all \
+	| awk -F" " '{ if ($$3 != "") print $$1 " " $$3; }' \
```
Comment (mvangent) [inline Makefile:15]: it's not working on my machine

` go list -m -u all | awk -F" " '{ if ($$3 != "") print $$1 " " $$3; }'

awk: illegal field $(), name "3"
 input record number 1, file
 source line number 1`

Comment (krysopath) [reply Makefile:15]: Mhhhm. That surprises me a bit. Can you run awk -h and posted it's output?

Comment (mvangent) [reply Makefile:15]: awk

usage: awk [-F fs] [-v var=value] [-f progfile | 'prog'] [file ...]

awk -h

awk: unknown option -h ignored

awk: no program given

**Review by mvangent — APPROVED** [review]
working like a charm now

**Pull request merged**